src/actions/helpers/index.ts

```diff
@@ -1,9 +1,27 @@
+import { makeApiCall } from "../../api/helpers";
 import { Registrar } from "../../contracts/types";
-
 import { getLogger } from "../../utilities";
+import {
+  ETHEREUM_API_ID,
+  ETHEREUM_PRICE_ENDPOINT,
+  ERC20_PRICE_ENDPOINT,
+} from "./constants";
+import { CoinGeckoRequestOptions, CoinGeckoResponse } from "../types";
 
 const logger = getLogger("actions:helpers");
 
+const createUri = (tokenAddress?: string): string => {
```

Comment:
If this is typed to only support coin gecko, can we name it as such? 

Alternatively, it could take a PriceProvider from an enum like 
```
enum PriceProvider {
CoinGecko: 0,
Provider2: 1
}
```